packages/webapp/src/containers/Animals/AnimalsFilter/styles.module.scss

```diff
@@ -15,8 +15,7 @@
 
 // Placeholder until the layout is completed and filter button can be appropriately placed next to searchbar
```

Comment:
I think this part of the layout is complete; I'll delete my placeholder comment next PR 🙂

Reply:
Oh! I appreciated this comment and forgot to remove... thank you!!